modules/update-upgrading-oc-adm-upgrade-recommend.adoc

```diff
@@ -1,24 +1,24 @@
 // Module included in the following assemblies:
 //
-// * updating/updating_a_cluster/updating-cluster-cli.adoc
+// * updating/updating_a_cluster/updating-cluster-cli.adoc [removed from assembly]
 
 :_mod-docs-content-type: PROCEDURE
 [id="update-upgrading-oc-adm-upgrade-recommend_{context}"]
 = Finding recommended update paths with oc adm upgrade recommend (Technology Preview)
 
 When updating your cluster, the `oc adm upgrade` command returns a list of the next available versions. You can use the `oc adm upgrade recommend` command to narrow down your suggestions and recommend a new target release before you launch your update.
 
-The `oc adm upgrade recommend` command is read-only and cannot alter the state of your cluster.  
+The `oc adm upgrade recommend` command is read-only and cannot alter the state of your cluster.
 
 :FeatureName: The `oc adm upgrade recommend` command
 include::snippets/technology-preview.adoc[]
 
 [NOTE]
 ====
-Your cluster does not need to be a Technology Preview-enabled cluster in order for you to use the `oc adm upgrade recommend` command. 
+Your cluster does not need to be a Technology Preview-enabled cluster in order for you to use the `oc adm upgrade recommend` command.
```

Comment:
Once we GA recommend, I don't think we need this notification, right? cc @wking

Reply:
Right, this whole file is getting a `[removed from assembly]` comment up on line 3.  I don't understand why we just flag files as removed from assemblies instead of just removing the file, but as far as this pull is concerned, this whitespace removal is just tidying up dead-code, and 🤷, that doesn't hurt or help anything